From 686a06ab9584ad4818e2746aeaa38df76d66d7dc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=A9rgio=20M=2E=20Basto?= <sergio@serjux.com>
Date: Mon, 11 Nov 2019 04:55:56 +0000
Subject: [PATCH 8/8] Fixup

filesavedialog.h:7:10: fatal error: private/qguiapplication_p.h: No such file or directory
---
 eidguiV2/filesavedialog.cpp | 8 --------
 eidguiV2/filesavedialog.h   | 1 -
 2 files changed, 9 deletions(-)

diff --git a/eidguiV2/filesavedialog.cpp b/eidguiV2/filesavedialog.cpp
index 6d92446..3240ad0 100644
--- a/eidguiV2/filesavedialog.cpp
+++ b/eidguiV2/filesavedialog.cpp
@@ -8,7 +8,6 @@
 
 FileSaveDialog::FileSaveDialog(QQuickItem *parent)
     : QQuickItem(parent)
-    , m_dlgHelper(init_helper())
     , m_modality(Qt::WindowModal)
     #if QT_VERSION >= QT_VERSION_CHECK(5, 8, 0)
     , m_options(QFileDialogOptions::create())
@@ -108,13 +107,6 @@ void FileSaveDialog::setNameFilters(QStringList nameFilters)
     }
 }
 
-QPlatformFileDialogHelper* FileSaveDialog::init_helper()
-{
-    return static_cast<QPlatformFileDialogHelper*>(
-                QGuiApplicationPrivate::platformTheme()->createPlatformDialogHelper(QPlatformTheme::FileDialog)
-                );
-}
-
 void FileSaveDialog::open()
 {
     if (!valid()) return;
diff --git a/eidguiV2/filesavedialog.h b/eidguiV2/filesavedialog.h
index 7577d21..0c5416e 100644
--- a/eidguiV2/filesavedialog.h
+++ b/eidguiV2/filesavedialog.h
@@ -4,7 +4,6 @@
 #include <QQuickItem>
 #include <QFileDialog>
 #include <QUrl>
-#include <private/qguiapplication_p.h>
 #include <QtGui/qpa/qplatformdialoghelper.h>
 #include <QtGui/qpa/qplatformtheme.h>
 
-- 
2.21.0

